src/jobs/management/commands/skater.py

```diff
@@ -37,7 +37,7 @@ def dot_to_json(file_in):
 
```

Comment:
please consider renaming the file in "try_[..]" so that we know is one of those files in which we try out new stuff